spec/integration/subset_spec.rb

```diff
@@ -36,4 +36,18 @@
     subset1 = TTFunk::Subset.for(font, :unicode)
     expect { subset1.encode }.to_not raise_error
   end
+
+  it 'maps final code 0xFFFF to glyph 0 in generated type 4 cmap' do
+    font = TTFunk::File.open test_font('DejaVuSans')
+
+    subset = TTFunk::Subset.for(font, :unicode)
+    subset.use(97)
+    cmap = TTFunk::File.new(subset.encode).cmap
+
+    # Unicode subsets only contain a single format 4 cmap subtable.
+    expect(cmap.tables.size).to eq(1)
+    format04 = cmap.tables.first
+    expect(format04.format).to eq(4)
+    expect(format04[0xFFFF]).to eq(0)
```

Comment:
I'm not sure if this is an adequate assertion. `0` is a default value.

Reply:
@pointlessone Good point; I changed the test to check for `format04.code_map[0xFFFF]` instead.